README.md

```diff
@@ -1,84 +1,30 @@
 # Adobe Experience Platform Edge Identity Mobile Extension
-
+[![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/edgeidentity.svg?logo=android&logoColor=white&label=edgeidentity)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/edgeidentity)
 
 ## About this project
 
-The Adobe Experience Platform Edge Identity is a mobile extension for the [Adobe Experience Platform SDK](https://github.com/Adobe-Marketing-Cloud/acp-sdks) and requires the `MobileCore` extension. This extension enables handling of user identity data from a mobile app when using the AEP Mobile SDK and the Edge Network extension.
+The Adobe Experience Platform Identity for Edge Network mobile extension enables identity management from your mobile app when using the [Adobe Experience Platform Mobile SDK](https://developer.adobe.com/client-sdks/documentation) and the Edge Network extension.
 
 
 ### Installation
 
-Integrate the Edge Identity extension into your app by including the following in your gradle file's `dependencies`:
-
-```gradle
-implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
-implementation 'com.adobe.marketing.mobile:edge:1.+'
-implementation 'com.adobe.marketing.mobile:core:1.+'
-implementation 'com.adobe.marketing.mobile:edgeconsent:1.+' // Recommended when using the setAdvertisingIdentifier API
-```
+Integrate the Adobe Experience Platform Edge Identity Mobile extension into your app by following the [getting started guide](Documentation/getting-started.md).
```

Comment:
since you mentioned the full name above, you can use the short name here: 
"Integrate the Identity for Edge Network mobile extension .."